src/core/bittorrent/session.h

```diff
@@ -120,6 +121,7 @@ namespace BitTorrent
         QVector<int> filePriorities; // used if TorrentInfo is set
         bool ignoreShareRatio;
         bool skipChecking;
+        QList<QNetworkCookie> cookies;
```

Comment:
Incorrect!
Cookies have nothing to add torrent to the session. Use DownloadManager::instance()->setCookiesFromUrl() directly.